Bottom: a3d432337bb68006c99bd7ec6bf63346765c0c05
Top:    2d8f6970d59cbce4cb58f13b050373bf8328e1d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-24 23:14:13 +0100

hid/gtk: Simplify viewport setup in PCBChanged()

This version still works ;)


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index be4f1db..8fd11ef 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1145,9 +1145,7 @@ PCBChanged (int argc, char **argv, Coord x, Coord y)
     return 0;
   RouteStylesChanged (0, NULL, 0, 0);
   ghid_port_ranges_scale ();
-  ghid_port_ranges_pan (0, 0, FALSE);
   ghid_zoom_view_fit ();
-  ghid_port_ranges_changed ();
   ghid_sync_with_new_layout ();
   return 0;
 }
